packages/sdk-metrics/src/Descriptor.ts

```diff
@@ -32,49 +32,81 @@ export enum InstrumentType {
 /**
  * An interface describing the instrument.
  */
-export interface InstrumentDescriptor {
+export interface Descriptor {
   readonly name: string;
   readonly description: string;
   readonly unit: string;
-  readonly type: InstrumentType;
   readonly valueType: ValueType;
 }
 
-export function createInstrumentDescriptor(
+/**
+ * An interface describing the instrument.
+ * @deprecated Please use {@link Descriptor} instead.
+ */
+export interface InstrumentDescriptor extends Descriptor {
+  /**
+   * The original instrument's type.
+   * @deprecated Any necessary information about a metric's properties is available in the data point.
+   */
+  readonly type: InstrumentType;
+}
+
+/**
+ * An interface describing an actual instrument or virtual instrument (created by views).
+ */
+export interface MetricDescriptor extends Descriptor {
+  /**
+   * The original instrument's descriptor.
+   */
+  readonly originalInstrumentType: InstrumentType;
```

Comment:
I'm not sure about the value of renaming the field. As far as I could tell, this is only used internally. I would find documentation on the `InstrumentDescriptor.type` about "the type of the original instrument" would be good. Would you mind expanding on the renaming?

Reply:
I think you're right; I initially used the rename to determine where the `InstrumentDescriptor`-type was still used, but it does not add much value anymore. I changed it back to `type` in [0312bae](https://github.com/open-telemetry/opentelemetry-js/pull/3520/commits/0312baeec653ed5e2d8e11c0bcc55094d7343be5). :slightly_smiling_face: